packages/explat-client-react-helpers/src/index.tsx

```diff
@@ -1,13 +1,26 @@
 /**
  * External dependencies
  */
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 
 /**
  * WordPress dependencies
  */
 import type { ExPlatClient, ExperimentAssignment } from '@automattic/explat-client';
 
+interface UseExperimentOptions {
```

Comment:
The name of the type should be `ExperimentOptions`. The type can be used also with the `<Experiment>` component, with no hooks involved.